projects/vdk-plugins/vdk-structlog/src/vdk/plugin/structlog/constants.py

```diff
@@ -63,3 +66,55 @@
 ) | set(STRUCTLOG_LOGGING_METADATA_JOB.keys())
 
 RECORD_DEFAULT_FIELDS = set(vars(logging.LogRecord("", "", "", "", "", "", "")))
+
+
+def parse_log_level_module(log_level_module):
```

Comment:
maybe we should move these to another file since this one is called constants 
maybe log_levels.py or something similar?

Reply:
Makes sense. Moved them to a new file: log_level_utils.